### PR TITLE
Directory restructure and GeoJSON processing

### PR DESCRIPTION
**Pull request by michaelpkuhn**
file_to_json.py:
- reads in all files in data_files directory, returns dictionary

dict_merge.py:
- added geojson processing
 - added merge geojson function
 - added flatten geojson function for GIS compatibility

directory restructure:
- moved scripts to src/ folder
- added tests to test/ folder
- added tracked json to test/resources/ folder
- updated file paths

requirements.txt:
- added firebase admin sdk
- added geojson library to verify geojson format

output:
- modified json.dump to eliminate whitespace and save space

main.py:
- added geo_ls argument for testing on smaller county file

test_jsons.py:
- modified test to validate geo_json format

GeoJSON merging added for GIS example visualizations
qgis_poverty_county

### CODE REVIEW

**Review by jordenk — APPROVED** [review]

**Comment by michaelpkuhn** [comment]
I'm running into an issue with geojson rn, not sure what changed. Looking into that.

**Comment by michaelpkuhn** [comment]
> I'm running into an issue with geojson rn, not sure what changed. Looking into that.

There are more zip codes not in the dictionary than I remember.

**Comment by michaelpkuhn** [comment]
Added panda implementation

File: src/dict_merge.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                        continue
+                    #Otherwise, raise exception and return metric name
+                    e = 'Duplicate metric key: rename to avoid overwrite'
+                    raise Exception(e, v)
+                #The metric is not in the dictionary, add it
+                else:
+                    mergedDict[k][v] = d[k][v]
+    return mergedDict
+
+def getGeoJson(fp = 'shape_files/ILgeojson.json', param=""):
```
Comment (jordenk) [inline src/dict_merge.py:102]: We should change these function names to snake_case. get_geo_json. A linter would enforce this.

**Comment by michaelpkuhn** [comment]
Test passed [here](https://github.com/Code-For-Chicago/greater-chicago-food-despository/actions/runs/612476133), but not above?

**Pull request merged**